src/evm.rs

```diff
@@ -137,7 +137,7 @@ impl EVM {
         to: &str,
         value: Option<U256>,
         data: Option<Vec<u8>>,
-    ) -> PyResult<()> {
+    ) -> PyResult<String> {
```

Comment:
I think this should be left as a byte array so that it can be converted with `int.frombytes(..., byteorder='big')`